test/fixtures/payload-helpers/features/steps/scripting_steps.rb

```diff
@@ -13,3 +13,7 @@
 When('The HTTP response header {string} equals {string}') do |header, value|
   Maze.check.equal value, $http_response[header]
 end
+
+When('The HTTP response header {string} is null') do |header|
+  Maze.check.nil $http_response[header]
+end
```

Comment:
Should this really be checking the header isn't there?

```suggestion
When('The HTTP response header {string} is not present') do |header|
  Maze.check.false($http_response.key(header))
end
```

Reply:
I did consider (and would prefer) naming naming it "is not present", but there was already a precedent in `the {word} {string} header is null`.  Will change the implementation though.

Reply:
OK — I think that step is also wrong since a HTTP header can only ever be a string so checking it's null doesn't make sense, but it's probably to late to change that

Reply:
Not super important for this case but something that would help with that kind of thing is adding the new step in a v7 release, then removing the old one in v8

That way everything can be migrated over without having to also update to a new major release with other breaking changes

I'm not really convinced this is really worth changing TBH — there's no way the header could be null without it being missing, so it's really only a pedantic thing

Reply:
Not a bad suggestion, although it comes with the risk that we forget to remove the unwanted one and carry arguably worse tech debt into a new major version (i.e. two steps that do the same thing).  I'm on board with ironing out pedantic issues, though - I am basically a professional pedant after all.